app.rb

```diff
@@ -163,10 +71,10 @@ def choose_option
     option = gets.chomp.to_i
 
     case option
-    when 1 then list_books
-    when 2 then list_people
-    when 3 then create_person
-    when 4 then create_book
+    when 1 then ListBook.new(@books, self).list_books
+    when 2 then ListPeople.new(@person, self).list_people
+    when 3 then CreatePerson.new(@person, self).create_person
+    when 4 then CreateBook.new(@books, self).create_book
     when 5 then create_rental
     when 6 then list_rental_for_a_person
     when 7 then stop_application
```

Comment:
- [ ] While creating rentals and after entering all the details the app shows `Rental created successfully.` and it quite so users is not able to list the rentals they created. Please make sure to fix this issue and app should not quit after rentals are created and users should be able to list them.